assignments/4. operators/loops.md

```diff
@@ -32,20 +64,44 @@ var amount = 0;
 "1 * 4 = 4"
 .... for all 100 results)
 
+```js
+    for (let r = 1; r <= 10; r++ ) {
+        for (let p = 1; p <= 10; p++) {
+            console.log(`${r} * ${p} = ${r*p}`);
+        }
+    }
+```
+
 5. 🎖Show the following output using one loop.
 ```js
 // 1, 2, 3, 4, 5
 // 6, 7, 8, 9, 10
 
 // Your code goes here
+for (let k = 1; k <= 10; k++) {
+    console.log(k);
+}
 ```
 
 6. 🎖Use a while loop to add up the numbers 1 to 20.
 ```js
 // Your code goes here
+let num = 0;
+let start = 0;
+while ( start < 20) {
+    num = ++num;
+    ++ start;
+}
 ```
 
 7. 🎖Use a while loop to print out the even number from 1 to 20. (You'll need Modulus for this. And an IF Statement.)
 ```js
 // Your code goes here
+    let num = 1;
+    while ( num <= 20 ) {
+        if ( num % 2 == 0) {
+            console.log(num);
+        }
+        ++num;
+    }
```

Comment:
It's also returning `21` it should not. 

<img width="361" alt="Screenshot 2019-06-22 at 12 19 53" src="https://user-images.githubusercontent.com/3122626/59960604-21afbe80-94e8-11e9-8fb0-a1662201ebff.png">